<launch>
    <group ns="mint">
    	<rosparam file="$(find motion_intention)/config/mint_config.yaml" />
    	<node name="history_handler" pkg="motion_intention" type="history_handler_node" />
    	<param name="intention_predictor/hparam_json_path" type="string" value="$(find motion_intention)/config/hyper_params.json" />
    	<param name="intention_predictor/model_weights_path" type="string" value="$(find motion_intention)/config/traced_mintnet_epochs_10.pt" />
    	<node name="mint" pkg="motion_intention" type="motion_intention_node" />
    </group>
</launch>
